File: prize/solutions/martinvl.py
# @EXPECTED_RESULTS@: CORRECT

from sys import stdin

stdin.next()

# count the number of prizes that are more valuable than the iphone
more_valuable = sum((float(line.split()[1]) > 5790 for line in stdin))

if more_valuable:
    print more_valuable + 1
else:
    print 'STEAL THE PHONE'

File: prize/solutions/martinvl.c
/*
@EXPECTED_RESULTS@: CORRECT
*/

#include <stdio.h>

int main(int argc, char* argv[])
{
    int n;
    scanf("%d", &n);

    int num_larger = 0;

    for (int i = 0; i < n-1; ++i) {
        double value;
        char buf[50];
        scanf("%s %lf", buf, &value);

        if (value > 5790) {
            ++num_larger;
        }
    }

    if (num_larger) {
        printf("%d\n", num_larger + 1);
    } else {
        printf("STEAL THE PHONE\n");
    }

    return 0;
}
